site/docs/reference/Connectors/capture-connectors/amazon-s3.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 2
+---
+# Amazon S3
+
+This connector captures data from an Amazon S3 bucket.
+
+[`ghcr.io/estuary/source-s3:dev`](https://ghcr.io/estuary/source-s3:dev) provides the latest connector image when using the Flow GitOps environment. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, either your S3 bucket must be public,
+or you must have access via a root or [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html).
+
+* For public buckets, verify that the [access policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-overview.html#access-control-resources-manage-permissions-basics) allows anonymous reads.
+* For buckets accessed by a user account, you'll need the AWS **access key** and **secret access key** for the user.
+See the [AWS blog](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) for help finding these credentials.
+
+## Configuration
+
+There are various ways to configure and implement connectors. See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about these methods. The values and code sample in this section provide configuration details specific to the Amazon S3 source connector.
```

Comment:
nits: 
- "and implement" seems unnecessary and confusing
- I think plain "example" is preferred over "code sample" since it feels a little weird to call yaml _code_